resources/assets/components/InboxItem/index.js

```diff
@@ -64,6 +64,7 @@ class InboxItem extends React.Component {
               {campaign['reportback_info']['noun']} {campaign['reportback_info']['verb']}
             </div>
           </article>
+          <a href="#" onClick={e => this.props.showHistory(post['id'], e)}>Edit | Show History</a>
```

Comment:
great question! This is the way that it's formatted in the mock-up and the whole thing appears to be just one link. They definitely go to the same place since editing the quantity and showing the history are both in this modal. this might be one for @lkpttn ?